setup.cfg

```diff
@@ -1,5 +1,6 @@
 [metadata]
 name = requests-unixsocket
+version = 0.2.1
```

Comment:
Pretty sure this is wrong: the project version is produced from Git tags via `pbr`. Besides, the version on PyPI is 0.3.0 already.

Reply:
Right, this was never intended to end up here. That's what I get for using master for a PR!

Reply:
/me just wanted to make a release in his fork, until a new release was made upstream.

I also did not expect it would take so long to actually land.

Reply:
Ah, makes sense. Have you checked if installing from Git URLs would work if you make a tag in your fork? Also, you could make a dumb-pypi index on GH Pages with a custom build and install from there.

FWIW I checked the maintainer's profile and it seems to be rarely used. So I wrote emailed them yesterday about this with suggestions. No reply so far.